lib/index.js

```diff
@@ -159,6 +165,27 @@ scrapeIt.scrapeHTML = ($, opts) => {
                         $elm = $elm.eq(cOpt.eq)
                     }
 
+                    if (typpy(cOpt.texteq, Number)) {
+                        let children = $elm.contents();
```

Comment:
👍  for `contents()`, I didn't know cheerio supports it. Will review this soon.

Reply:
I am not sure whether texteq should also support deep-searching for text nodes from an API standpoint. It would be useful in cases where the actual markup changes often: `<div>Blah</div>`, `<div><b>Blah</b></div>`. Just because it got surrounded with a `<b>`, it shouldn't throw scrape-it off its track.